doc/toc.md

```diff
@@ -4,6 +4,8 @@
 ## [2.1](release-notes/2.1.md)
 ## [2.0](release-notes/2.0.md)
 ## [1.1](release-notes/1.1.md)
+# Creating a Model
+## [Indices](modeling/indices.md)
```

Comment:
As above with the naming. 

To clarify, either would be fine with me, as long as we're consistent and following how PostgreSQL uses these word-versions.

Reply:
Great catch, thanks...